File: frontend-app/lib/features/merchant/module.dart
import 'package:go_router/go_router.dart';
import 'package:shopping_app/app/module.dart';
import 'package:shopping_app/features/merchant/presentation/merchant_home_page.dart';
import 'package:shopping_app/features/merchant/presentation/merchant_staff_page.dart';
import 'package:shopping_app/features/merchant/presentation/merchant_stats_page.dart';

final merchantModule = AppModuleContribution(
  key: 'merchant',
  owner: '成员 2',
  routes: [
    GoRoute(
      path: '/merchant',
      builder: (context, state) => const MerchantHomePage(),
    ),
    GoRoute(
      path: '/merchant/staff',
      builder: (context, state) => const MerchantStaffPage(),
    ),
    GoRoute(
      path: '/merchant/stats',
      builder: (context, state) => const MerchantStatsPage(),
    ),
  ],
);

File: frontend-miniapp/miniprogram/features/merchant/module.ts
import type { AppModuleContribution } from '../../app/module-types'

export const merchantModule: AppModuleContribution = {
  key: 'merchant',
  owner: '成员 2',
  pages: [
    '/package-merchant/pages/home/index',
    '/package-merchant/pages/products/index',
    '/package-merchant/pages/add-product/index',
    '/package-merchant/pages/staff/index',
    '/package-merchant/pages/stats/index',
    '/package-merchant/pages/inbox/index',
  ],
}
